### PR TITLE
Forgot to branch Hopefully nothing breaks 

### PR DESCRIPTION
**Pull request by Soniapotato**
Added 3/4 components, 

please review: 
 @gbb1 @bxandrew @PHANTOMS0308

### CODE REVIEW

File: src/components/qa/QuestionsAnswers.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, {useState, useEffect} from 'react';
+import QuestionList from './QuestionList.jsx';
+import Search from './Search.jsx';
+
+const QuestionsAnswers = ()  => (
+  <section>
+  <div id='question-answers'>
```
Comment (PHANTOMS0308) [inline src/components/qa/QuestionsAnswers.jsx:7]: forget to indent, but it's fine, you could fix it later

File: src/components/qa/Search.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React, {useState, useEffect} from 'react';
+
+const Search = () => {
+
+  return (
+  <section id='search-q-a'>
```
Comment (PHANTOMS0308) [inline src/components/qa/Search.jsx:6]: Same indentation here, ha

**Review by gbb1 — APPROVED** [review]
Looks good!

**Review by PHANTOMS0308 — COMMENTED** [review]
It's all good.

File: src/components/qa/QuestionsAnswers.jsx (hunk 1 above)
Comment (Soniapotato) [reply src/components/qa/QuestionsAnswers.jsx:7]: Good catch

**Pull request merged**